### ISSUE TITLE
How to make the config.json available from a mount path inside the container?

### ISSUE THREAD

**Issue by SmartPhoneLover** [monitoror/monitoror#402]
Hello,

I'm stuck in the part of passing the 'config.json' file (first launch), but I can't manage to do that. I'm not really sure what path to mount to allow Monitoror to read the JSON.

I'm trying with the VARIABLE=MO_CONFIG as '/mnt/user/appdata/monitoror/config.json' (the path of one of my server shares).
_default -> /mnt/user/appdata/monitoror/config.json_

Note: There's no problem if I share the file using my NextCloud instance. But I would like the file to be read locally, using a path instead.

Thank you.

![Captura](https://user-images.githubusercontent.com/42898749/144603161-664be3da-124c-4d68-a028-56da96164877.JPG)
![34346](https://user-images.githubusercontent.com/42898749/144603475-54e95d35-217d-482b-a52f-3af143410e20.JPG)

**Comment by Alex-D** [comment]
Hey,
Does it work with a non-mounted file?

**Comment by SmartPhoneLover** [comment]
> Hey, Does it work with a non-mounted file?

Works, but the message to proceed with configuration of the file (copy paste sample) appears.

**Comment by Alex-D** [comment]
So, it does not work. Seems like your env variable is now set correctly :/

**Comment by SmartPhoneLover** [comment]
> So, it does not work. Seems like your env variable is now set correctly :/

Mmm, I think I missundertood what you have asked before.

I wanted to mean that, if I don't set the -v param, Monitoror just tells that I need to create the config.json file and place it near the monitord. But, what is the path inside the container where it's located, so I can mount it to be persistent so I can place the file there, as suggested?

Again, if I set the 'MO_CONFIG' variable with a HTTPS address that includes the config file, it works, obviously. Perfect.
But, I would like the file to be placed inside a mapped location, so I can edit it more accordingly.

**Comment by Alex-D** [comment]
The MO_CONFIG must point to the file path *inside* the container, relative to the Monitoror container itself, not the host.
So, if you add a volume, use the path *inside* the container, not the one on the host. 
Am I missing something?

**Comment by SmartPhoneLover** [comment]
> The MO_CONFIG must point to the file path _inside_ the container, relative to the Monitoror container itself, not the host. So, if you add a volume, use the path _inside_ the container, not the one on the host. Am I missing something?

Yeah, that is what I would like to do. I mean, mount the corresponding internal data from the container where that config file should be located. But, what is the internal path? For other containers are, example:
- /data/config
- /user
- etc...

**Comment by Alex-D** [comment]
Hum, the container is based on Alpine, so you can do what you want, there is no default path or so.
Monitoror's Dockerfile can be seen here: https://github.com/monitoror/monitoror/blob/develop/Dockerfile

**Comment by SmartPhoneLover** [comment]
I finally solved it by:
1. Create a mount path pointing to a location on my host.
2. Place the config.json file on the host path from before.
3. Add the var 'MO_CONFIG', and set it as accordingly to the path previously set.

**-v '/mnt/user/appdata/monitoror/':'/configs':'ro'**
Here I make the content of '.../monitoror/*' from host to be shown from inside container somewhere. In that case I decided to place it in the root. Now I can place the file under the corresponding dir on my host.
**-e 'MO_CONFIG'='/configs/config.json'**
Here I create the corresponding variable, but pointing to the virtual directory where the file is now avilable. Now works.

Thank you for your help.
